account_chart_update/wizard/wizard_chart_update.py

```diff
@@ -1079,24 +1079,25 @@ def _update_taxes_pending_for_accounts(self, todo_dict):
         when the referenced accounts are still not available).
         """
         done = self.env["account.tax"]
-        for k, v in todo_dict["account_dict"]["account.tax"].items():
+        for tax, tpl_account_dict in todo_dict["account_dict"]["account.tax"].items():
```

Comment:
Please don't rename these variables (or do it in a separate commit as `[REF]`) for focusing the patch on the important thing.

Reply:
I removed the renaming of `v` variable, but kept the k renaming since it's closely linked to the fix : otherwise I would need to rename the `tax` variable to `k` a few lines above